app/scheduler/ProgressToSiftJob.scala

```diff
@@ -34,13 +37,26 @@ trait ProgressToSiftJob extends SingleInstanceScheduledJob[BasicJobConfig[Waitin
   val batchSize = conf.batchSize.getOrElse(throw new IllegalArgumentException("Batch size must be defined"))
 
   def tryExecute()(implicit ec: ExecutionContext): Future[Unit] = {
+    implicit val hc: HeaderCarrier = new HeaderCarrier()
     siftService.nextApplicationsReadyForSiftStage(batchSize).flatMap {
       case Nil => Future.successful(())
       case applications => siftService.progressApplicationToSiftStage(applications).map { result =>
+        result.successes.map { application =>
+          if (isSiftEnteredStatus(application)) {
+            siftService.sendSiftEnteredNotification(application.applicationId).map { unit =>
+              play.api.Logger.info(s"Notified ${application.applicationId} about SIFT_ENTERED")
```

Comment:
@WarFox 👀

Reply:
removing this log, as email can be verified at the mailgun